lambdas/GitHubAccessCodeGetter/index.js

```diff
@@ -3,12 +3,39 @@
 console.log('Loading GitHubAccessCodeGetter lambda function');
 
 const axios = require('axios');
+const _ = require('lodash');
+
+const clientId = process.env.CLIENT_ID;
+const clientSecret = process.env.CLIENT_SECRET;
+
+const authUrl = (token) =>
+`https://api.github.com/applications/${clientId}/tokens/${token}`
+const authOpts = {
+  auth: {
+    username: clientId,
+    password: clientSecret
+  }
+}
+const orgUrl = `https://api.github.com/user/orgs`
+const orgOpts = (token) => ({
+  headers: {
+    'Authorization': `token ${token}`
+  }
+})
+
+// Create whitelist array, after checking for the env var
+if (!process.env.Whitelist) console.err('Whitelist env var missing!')
```

Comment:
Good point, will fix. This is actually a problem with some of our other Lambda's env vars, they don't all adhere to the same naming convention.